src/worker/queues/sendWebhookQueue.ts

```diff
@@ -88,3 +115,36 @@ const getContractSubscriptionWebhookIdempotencyKey = (args: {
   }
   throw 'Must provide "eventLog" or "transactionReceipt".';
 };
+
+/**
+ * Transaction webhooks
+ */
+
+const enqueueTransactionWebhook = async (
+  data: EnqueueTransactionWebhookData,
+) => {
+  if (!_queue) return;
+
+  const webhooks = [
+    ...(await getWebhooksByEventType(WebhooksEventTypes.ALL_TX)),
+    ...(await getWebhooksByEventType(data.type)),
+  ];
```

Comment:
Get all webhooks that match the exact type or `ALL_TX` which cares about all event types.